seatunnel-core/seatunnel-starter/src/main/java/org/apache/seatunnel/core/starter/seatunnel/args/ClientCommandArgs.java

```diff
@@ -49,6 +49,14 @@ public class ClientCommandArgs extends AbstractCommandArgs {
         description = "Get job status by JobId")
     private String jobId;
 
+    @Parameter(names = {"-can", "--cancel-job"},
+        description = "Cancel job by JobId")
+    private String cancelJobId;
+
+    @Parameter(names = {"-m", "--metrics"},
```

Comment:
@Hisoka-X @EricJoy2048 Hello, The latest code `-m` parameter has been used. Does this PR still need to be done? What should I do?

Reply:
how about use `-d`? mean deploy mode.